src/applications/mhv-secure-messaging/tests/e2e/pages/FolderLoadPage.js

```diff
@@ -64,6 +64,7 @@ class FolderLoadPage {
     this.loadFolderMessages('Drafts', -2, 1, messagesList);
   };
 
+  // this method no longer needed as sent folder link was moved to nav-bar
   loadSentMessages = (messagesList = mockSentMessages) => {
     this.loadFolderMessages('Sent', -1, 2, messagesList);
   };
```

Comment:
If this is no longer needed, maybe we can 86 this? cc: @fazilqa

Reply:
I prefer to keep it for a bit more time and will remove it in my next cleanup PR

Reply:
I see this is already merged. @fazilqa I believe we may want to discard any code in the future that is not planned to be used. we can always recover it from git version history 👍🏻